docker-compose.yml

```diff
@@ -169,9 +169,10 @@ services:
     volumes:
       - ./projects/minimal_operateur_server/:/app
       - minimal-operateur-server-settings:/settings
+      - ./scripts/wait-for-redis.sh:/wait-for-redis.sh
     depends_on:
       - redis
-    command: ['./wait-for-it.sh', 'redis:6379', '--', 'flask', 'rq', 'worker']
+    command: ['/wait-for-redis.sh', 'redis', '6379', 'flask', 'rq', 'worker']
```

Comment:
Tempting to hard code the host and port, and make the wrapper even simpler.

Reply:
I prefer not to:

- docker-compose defines the services and their names
- the script doesn't have to know where redis is hosted

it's a detail, but imo it's better to leave it as-is for the separation of concerns